tests/buttons/tests/01-run.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2017 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import sys
+
+sys.path.append(os.path.join(os.environ['RIOTBASE'], 'dist/tools/testrunner'))
+import testrunner
+
+def testfunc(child):
+    child.expect_exact("On-board button test")
+    index = child.expect([
```

Comment:
You don't check `index` anymore, please remove assignment

Reply:
now I do use it :-)